### PR TITLE
fixed panel formatiing for dashboard.js

### PR DESCRIPTION
**Pull request by 602dhruviii**
# Description
Summarize the change.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: static/js/metrics-explorer.js [hunk 1]
```diff
@@ -2008,7 +2013,7 @@ async function getMetricNames() {
     };
     const res = await $.ajax({
         method: 'post',
-        url: 'metrics-explorer/api/v1/metric_names',
+        url: 'http://playground.siglens.com:5122/metrics-explorer/api/v1/metric_names',
```
Comment (sonamgupta21) [inline static/js/metrics-explorer.js:2016]: Remove playground url

Comment (602dhruviii) [reply static/js/metrics-explorer.js]: done

File: static/metrics-ingestion.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
         <div class="myOrg">
             <div class="myOrg-container">
                 <div class="ingestion-nav-tab">
-                    Metrics Ingestions
+                    Metrics Ingestions Methods
```
Comment (sonamgupta21) [inline static/metrics-ingestion.html:80]: It should be `Ingestion`

File: static/traces-ingestion.html [hunk 3]
```diff
@@ -77,7 +77,7 @@
         <div class="myOrg">
             <div class="myOrg-container">
                 <div class="ingestion-nav-tab">
-                    Traces Ingestions
+                    Traces Ingestions Methods
```
Comment (sonamgupta21) [inline static/traces-ingestion.html:80]: Same here.

**Pull request merged**